tests/api/test_instance.py

```diff
@@ -66,14 +67,20 @@ def test_instances_status_marathon(
     assert response['marathon']['desired_state'] == 'start'
 
 
-@mock.patch('paasta_tools.api.views.instance.chronos_tools.load_chronos_config', autospec=True)
+@mock.patch('paasta_tools.chronos_tools.load_chronos_config', autospec=True)
 @mock.patch('paasta_tools.api.views.instance.chronos_tools.get_chronos_client', autospec=True)
-@mock.patch('paasta_tools.api.views.instance.chronos_tools.load_chronos_job_config', autospec=True)
+@mock.patch('paasta_tools.chronos_tools.load_chronos_job_config', autospec=True)
 @mock.patch('paasta_tools.api.views.instance.validate_service_instance', autospec=True)
 @mock.patch('paasta_tools.api.views.instance.get_actual_deployments', autospec=True)
-@mock.patch('paasta_tools.api.views.instance.select_tasks_by_id', autospec=True)
-@mock.patch('paasta_tools.api.views.instance.get_cached_list_of_running_tasks_from_frameworks', autospec=True)
+@mock.patch('paasta_tools.chronos_serviceinit.select_tasks_by_id', autospec=True)
+@mock.patch('paasta_tools.chronos_serviceinit.get_cached_list_of_running_tasks_from_frameworks', autospec=True)
+@mock.patch('paasta_tools.chronos_serviceinit.chronos_tools.lookup_chronos_jobs', autospec=True)
+@mock.patch('paasta_tools.chronos_serviceinit.chronos_tools.get_job_type', autospec=True)
+@mock.patch('paasta_tools.chronos_serviceinit.chronos_tools.get_chronos_status_for_job', autospec=True)
 def test_chronos_instance_status(
```

Comment:
You should just mock `paasta_tools.chronos_serviceinit.status_chronos_jobs`, assert that it's called correctly, and delete the other assertions here. You don't need to test the implementation of chronos_serviceinit (e.g. that it calls lookup_chronos_jobs and stuff) because the chronos_serviceinit tests should cover that already.